### DISCUSSION TITLE
Is there any way to disable copying the whole paragraph in case nothing is selected?

### DISCUSSION THREAD

**Discussion by SN-CH** [Zettlr/Zettlr#4531] [Q&A] [answered]
By default, Zettlr (or, probably, some third-party component) is configured to copy the whole paragraph if Ctrl-C is hit while nothing is selected.

It is clearly a useful feature, but in my specific case it makes life harder (some automation scripts fail because of this). Is there any way to turn it off (I mean the clipboard content does not change unless something is selected)? It seems there is no way to do it via GUI (and I am not asking to implement it, most likely I have a fringe case). But can it be done with config files or something?

**Comment by nathanlesage** [comment, marked as answer]
Ctrl-C copying the entire line when no selection is happening is a function that I've activated for the editor because for tons of cases it is extremely useful. I think it is too expensive to enable or disable individual functions at this time, however, so I don't know if we could ever selectively disable keyboard shortcuts.

**Comment by SN-CH** [comment]
I see, that sounds reasonable. Like I said I have a special case. Anyway, thank you for your answer.
